winappdbg/debug.py

```diff
@@ -505,7 +507,8 @@ def execl(self, lpCmdLine, **kwargs):
             # Warn when mixing 32 and 64 bits.
             # This also allows the user to stop attaching altogether,
             # depending on how the warnings are configured.
-            if System.bits != aProcess.get_bits():
+            BITS_WARNING_FLAG = False

```

Comment:
I'm curious about this, why remove the warning?

Reply:
Sorry about that, it was for my project and i forgot to remove this flag